README.rst

```diff
@@ -129,3 +129,25 @@ Treats ``os.environ("ALLOWED_HOSTS")`` as a comma-separated list.
 ~~~~~~~~~~~~~~
 
 Uses ``os.environ("SECRET_KEY)`` - required if ``DEBUG==False``.
+
+
+Custom Settings
+---------------
+
+You can make ``django12factor`` load arbitrary settings from environment variables with the ``custom_setting`` ``kwarg``.
+This takes an iterable of ``django12factor.EnvironmentVariableLoader`` instances (also importable as ``django12factor.EVL`` for brevity) or strings
+
+``EnvironmentVariableLoader`` instances are created with a ``name`` - the name of the environment variable to load, an optional ``default`` for the case when the environment variable is not set, and an optional ``loader`` - a callable that should take the string value of an environment variable and convert it as desired for your settings.
```

Comment:
Perhaps throw some kind of exception or warning if people use `parser=bool` too, since it's almost certainly not going to do what they want. Or, in the spirit of [heroku-fucking-console](https://github.com/tpope/heroku-fucking-console), just use `getenv_bool` if they pass in a bool.